hazelcast/src/main/java/com/hazelcast/spi/impl/operationparker/impl/ParkedOperation.java

```diff
@@ -34,15 +36,18 @@
 
 import static com.hazelcast.util.EmptyStatement.ignore;
 
-class ParkedOperation extends Operation implements Delayed, PartitionAwareOperation {
-    final Queue<ParkedOperation> queue;
-    final Operation op;
-    final BlockingOperation blockingOperation;
-    final long expirationTime;
+public class ParkedOperation extends Operation implements Delayed, PartitionAwareOperation, IdentifiedDataSerializable {
```

Comment:
ParkedOperation is a local operation and will never be serialized. So it doesn't need to implement any form of serialization. This should be added to the documentation to prevent confusion